tailormap-components/projects/core/src/lib/feature-form/form-creator/form-creator.component.ts

```diff
@@ -66,6 +66,8 @@ export class FormCreatorComponent implements OnChanges, OnDestroy, AfterViewInit
   private destroyed = new Subject();
 
   public ngOnChanges() {
+    document.documentElement.style.setProperty('--overlay-panel-columns', Math.max.apply(Math,this.formConfig.fields.map(field=> field.column)));
```

Comment:
I think its more efficient to create a Column model or something. So you only have to create the columns once and you could also add the attributes to this column model so you don't need the getAttributes method below.